apps/hmc/hmc-cft-hearing-service/perftest.yaml

```diff
@@ -6,10 +6,14 @@ spec:
   releaseName: hmc-cft-hearing-service
   values:
     java:
-      memoryLimits: "3Gi"
-      memoryRequests: "2Gi"
-      cpuLimits: "2000m"
-      cpuRequests: "500m"
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 4
+      memoryRequests: "1536Mi"
```

Comment:
This would not allow to scale down pods if it bumped to 3 or 4.. I think, this should be 2Gb. And even then scaling down to 2 pods might not happen if memory usage increased by only 50mb or so